assets/in

```diff
@@ -69,11 +69,11 @@ if test "$depth" -gt 0 2> /dev/null; then
 fi
 
 tagflag=""
-if [ -d $tag_filter ]; then
+if [ -n "$tag_filter" ]; then
   tagflag="--tags"
 fi
 
-git clone $depthflag $uri $branchflag $destination $tagflag
+git clone --single-branch $depthflag $uri $branchflag $destination $tagflag
```

Comment:
note to future self: this just undoes part of https://github.com/concourse/git-resource/commit/a62caf6dfdc8bd5f5bd579d85b48817cd980145e since it wasn't necessary (comment here: https://github.com/concourse/git-resource/pull/225#discussion_r234826904)